### PR TITLE
baresip, rem & re: update to 2.7.0.

### PR DESCRIPTION
**Pull request by index-0**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-musl)
- I built this PR locally for these architectures:
  - armv6l

### CODE REVIEW

**Comment by classabbyamp** [comment]
this should be one commit per package (so 3 total)

**Comment by index-0** [comment]
All the packages are related, and I was afraid that making separate commits would cause problems.

**Comment by classabbyamp** [comment]
just order them in a way that makes sense, separate commits won't cause issues

**Comment by index-0** [comment]
It's done

File: srcpkgs/re/patches/bsdnet.diff [hunk 1]
```diff
@@ -1,29 +1,29 @@
---- a/src/net/posix/pif.c	2016-12-03 15:21:22.000000000 +0100
-+++ b/src/net/posix/pif.c	2016-12-03 15:21:39.000000000 +0100
+--- a/src/net/ifaddrs.c
```
Comment (sgn) [inline srcpkgs/re/patches/bsdnet.diff:1]: I would remove this patch completely and add:
```
CFLAGS=-D_GNU_SOURCE
```
in template.

File: srcpkgs/re/template [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
 makedepends="openssl-devel zlib-devel"
 short_desc="Portable library for real-time communications"
 maintainer="John <me@johnnynator.dev>"
 license="BSD-3-Clause"
 homepage="https://github.com/baresip/re/"
-distfiles="https://github.com/baresip/re/archive/v${version}.tar.gz"
-checksum=43aa439b96aff75fe5768b9f9d49dea97042e42e7647df47b345465763e2f7ed
+distfiles="https://github.com/baresip/re/archive/refs/tags/v${version}.tar.gz"
+checksum=999f02b4299f9f4bbf637cf610099b656225fef0ce08ce56728978214d448343
 shlib_provides="libre.so"
```
Comment (sgn) [inline srcpkgs/re/template:19]: I think thisline should be removed now, and move `libre.so` to `-devel`.

File: srcpkgs/rem/template [hunk 3]
```diff
@@ -14,8 +14,8 @@ short_desc="Library for real-time audio and video processing"
 maintainer="John <me@johnnynator.dev>"
 license="BSD-3-Clause"
 homepage="https://github.com/baresip/rem"
-distfiles="https://github.com/baresip/rem/archive/v${version}.tar.gz"
-checksum=bcc91bb521fae183357fb422b00a3981477a22e99d3afe165c4ec50a6bbed9da
+distfiles="https://github.com/baresip/rem/archive/refs/tags/v${version}.tar.gz"
+checksum=7f4ad245eb9fb07a82890022d1c721c93ee2e299acb88a7e775ca541f7a09892
 shlib_provides="librem.so"
```
Comment (sgn) [inline srcpkgs/rem/template:19]: Remove this line, too. And move it into devel

**Comment by sgn** [comment]
Please merge 2 last changes into 1.

**Comment by index-0** [comment]
Are the commits good? Or should I change something?

File: srcpkgs/rem/template (hunk 3 above)
Comment (sgn) [reply srcpkgs/rem/template]: No, I meant different things. I will do it instead.

**Comment by sgn** [comment]
This PR was never tested. baresip is moved to gtk+3.

**Pull request closed without merging**